cosmos/azure-spring-boot-starter-cosmos/cosmos-multi-database-single-account/pom.xml

```diff
@@ -22,7 +22,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-spring-data-cosmos</artifactId>
-      <version>3.12.0</version> <!-- {x-version-update;com.azure:azure-spring-data-cosmos;dependency} -->
+      <version>3.13.1</version> <!-- {x-version-update;com.azure:azure-spring-data-cosmos;dependency} -->
```

Comment:
Same here.

Reply:
These two samples depend on azure-spring-data-cosmos directly,
in my concern, our bom is focus on starters,binders,etc. dependencies.

if we use bom instead, for these samples only, I don't it is necessary.

but for the entire azure-spring-boot-samples, I agree with you, so we can give our customer the same experience with other samples..

Reply:
I think we can use another PR to address the issue, also we need to go through all samples to check other samples which can also import boms and test the samples .